### DISCUSSION TITLE
Creation of Wiki

### DISCUSSION THREAD

**Discussion by wizard-28** [pacstall/pacstall#355] [Ideas]
What pages shall the Pacstall wiki have?

I'm thinking a section of the main Pacstall program, and another section for the pacscript programming.

**Comment by wizard-28** [comment]
A FAQ section under the Pacstall section would be good.

**Reply by Elsie19** [reply]
Yes. I concur

**Reply by Elsie19** [reply]
I shall forthwith hither to thine creation of said FAQ

**Reply by wizard-28** [reply]
You are supposed to assign yourself to the linked issue first.

I already did it for you now

https://github.com/pacstall/pacstall/issues/356

**Reply by wizard-28** [reply]
How about "What are pacscripts"

**Comment by wizard-28** [comment]
It turns out that GitHub doesn't have pull request support for Wikis yet. So people have devised certain [alternative methods](https://stackoverflow.com/questions/10642928/how-to-pull-request-a-wiki-page-on-github)

Useful info: [GitHub documentation on editing wikis](https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages)

---

Or we could switch to some other wiki creation site instead of using GitHub's... Any ideas?

**Reply by Elsie19** [reply]
We could use gitbook

**Reply by wizard-28** [reply]
[GitBook](http://gitbook.com/) and [ReadTheDocs](https://readthedocs.org/) seem to be the most popular options.

**Comment by Elsie19** [comment]


**Reply by wizard-28** [reply]
Umm... You made a separate comment instead of a reply.

**Comment by wizard-28** [comment]
How about the ones stored in the "Todo" section [here](https://github.com/pacstall/pacstall/projects/5)

**Comment by D-Brox** [comment]
The multiple ways to install packages (choosing repo, local install, prompless and keep etc)

**Reply by wizard-28** [reply]
You mean a usage section?

**Reply by D-Brox** [reply]
Sort of, yes
